enclave/Cargo.lock

```diff
@@ -2640,6 +2641,7 @@ dependencies = [
  "derive_more",
  "env_logger",
  "frame-support",
+ "hex",
```

Comment:
We already have this library in the worker. At the moment only required for test fixtures.